content/docs/12-enterprise-version/03-deploying-palette-with-helm.md

```diff
@@ -0,0 +1,326 @@
+    ```
+
+    Change `--region` and `--nodes` as required. You can also change the instance size, but keep in mind the required minimum of three nodes [with an instance](https://aws.amazon.com/ec2/instance-types/) with at least 4 CPUs and 12 GB of Memory.
+
+
+3. Once the cluster is available, go ahead and configure the OpenID Connect (OIDC) for the cluster to use Palette as the Identity Provider (IDP).
+
+    <br />
+
+    ```shell
+    eksctl utils associate-iam-oidc-provider --cluster=palette-selfhost --approve
+    ```
+
+4. Next, add the EBS Container Storage Interface (CSI) driver IAM role. Replace the `<AWS_ACCOUNT_ID>` with your AWS account ID.
+
+    <br />
+
+    ```shell
+    eksctl create addon --name aws-ebs-csi-driver \ 
+     --cluster palette-selfhost \ 
+     --service-account-role-arn arn:aws:iam::<AWS_ACCOUNT_ID>:role/AmazonEKS_EBS_CSI_DriverRole \
+     --force
+    ```
+
+5. Log in to the [AWS console](console.aws.amazon.com) and navigate to the EKS Dashboard.
+
+
+
+6. Select the cluster **palette-selfhost** to access its details page. 
+
+
+
+7. From the cluster details page, click on **Compute** > **Node Group**. Next, click on **Node IAM role ARN link**. 
+
+    ![A view of the cluster details page with the Node IAM role ARN highlighted](/enterprise-version_deploying-palette-with-helm_aws-iam-role.png)
+
+
+8. From the **Permissions** tab, click on the **Add Permissions** button, and select **Attach Policies**.
+
+
+9. Search for the policy **AmazonEBSCSIDriverPolicy** and add it to the role. 
+
+    <br />
+
+    <InfoBox>
+
+    You can find additional guidance about Amazon EBS CSI drivers and requirements by reviewing the [EBS User Guide](https://docs.aws.amazon.com/eks/latest/userguide/ebs-csi.html) and the [Manage EBS with EKS](https://github.com/awsdocs/amazon-eks-user-guide/blob/master/doc_source/managing-ebs-csi.md) guide.
+
+    </InfoBox>
+
+10. Extract the Helm Chart files from the compressed asset you received from us. Replace the file path as needed, and the version place holder.
+
+    <br />
+
+    ```shell
+    tar xzvf path/to-file/spectro-mgmt-helm-charts-X.X.tar.gz 
+    ```
+
+11. Navigate to the **spectro-mgmt-helm-charts-X.X** folder.
+
+    <br />
+
+    ```shell
+    cd spectro-mgmt-helm-charts-X.X
+    ```
+
+12. Review the **values.yaml** .You must populate the parameter `env.rootDomain` to the custom domain you will use for the installation. In addition, add the same `rootDomain` with port `:4222` to the `natsUrl` in the `nats` section of the YAML. Example: `env.rootDomain: my-domain.com:4222`
+
+    All other parameter values are optional and can be reset or changed with a Helm upgrade operation.
+
+    <br />
+    
+ 13. If you wish to use [AWS ACM for SSL Certs](https://docs.aws.amazon.com/acm/latest/userguide/acm-overview.html), instead of the default self-signed certificate generated by the Nginx *ingress controller*, you can add it to the `annotations` under `ingress`.
```

Comment:
```suggestion
 13. If you wish to use [AWS ACM for SSL Certs](https://docs.aws.amazon.com/acm/latest/userguide/acm-overview.html), instead of the default self-signed certificate that the Nginx *ingress controller* generates, you can add it to the `annotations` under `ingress`.
```